products/datacatalog/api.yaml

```diff
@@ -72,6 +72,202 @@ objects:
+      (such as a BigQuery dataset or a Pub/Sub topic) or outside of Google Cloud Platform. Clients can use
+      the linkedResource field in the Entry resource to refer to the original resource ID of the source system.
+
+      An Entry resource contains resource details, such as its schema. An Entry can also be used to attach
+      flexible metadata, such as a Tag.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': https://cloud.google.com/data-catalog/docs
+      api: https://cloud.google.com/data-catalog/docs/reference/rest/v1/projects.locations.entryGroups.entries
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: entryGroup
+        required: true
+        url_param_only: true
+        input: true
+        description: |
+          The name of the entry group this entry is in.
+      - !ruby/object:Api::Type::String
+        name: entryId
+        required: true
+        url_param_only: true
+        input: true
+        description: |
+          The id of the entry to create.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The Data Catalog resource name of the entry in URL format.
+          Example: projects/{project_id}/locations/{location}/entryGroups/{entryGroupId}/entries/{entryId}.
+          Note that this Entry and its child resources may not actually be stored in the location in this name.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: linkedResource
+        description: |
+          The resource this metadata entry refers to.
+          For Google Cloud Platform resources, linkedResource is the full name of the resource.
+          For example, the linkedResource for a table resource from BigQuery is:
+          //bigquery.googleapis.com/projects/projectId/datasets/datasetId/tables/tableId
+          Output only when Entry is of type in the EntryType enum. For entries with userSpecifiedType,
+          this field is optional and defaults to an empty string.
+      - !ruby/object:Api::Type::String
+        name: displayName
+        description: |
+          Display information such as title and description. A short name to identify the entry,
+          for example, "Analytics Data - Jan 2011".
+      - !ruby/object:Api::Type::String
+        name: description
+        description: |
+          Entry description, which can consist of several sentences or paragraphs that describe entry contents.
+      - !ruby/object:Api::Type::String
+        # This is a string instead of a NestedObject because schemas contain ColumnSchemas, which can contain nested ColumnSchemas.
+        # We'll have people provide the json blob for the schema instead.
+        name: schema
+        description: |
+          Schema of the entry (e.g. BigQuery, GoogleSQL, Avro schema). An entry might not have any schema attached to it.
```

Comment:
Do we want to add somewhere in the description that we're expecting the json blob? Or is it obvious? Maybe even a link to what the schema might look like?